main.tf

```
@@ -104,3 +104,20 @@ data "google_iam_policy" "policy" {
     }
   }
 }
+
+resource "google_project_iam_audit_config" "project" {
+  for_each = var.module_enabled ? var.audit_logs : []
+
+  project = var.project
+
+  service = each.value.service
+
+  dynamic "audit_log_config" {
+    for_each = each.value.config
```

we should keep the original naming: 

```suggestion
    for_each = each.value.audit_log_configs
```